bin/lint_go

```diff
@@ -6,4 +6,9 @@ if [ -n "$(gofmt -d cmd lib 2>&1)" ]; then
   exit 1
 fi
 
-gometalinter.v1 --deadline=20s
+# Disabled linters:
+# - https://github.com/GoASTScanner/gas: not production-ready at this point
+# - gocyclo: will be enabled and the code refactored later
+# - goconst: let's enable this after more generic refactorings
+gometalinter.v1 -D gas -D gocyclo -D goconst lib/...
+gometalinter.v1 cmd --deadline=20s
```

Comment:
Good catch, I have removed it. This still runs on my Macbook. This is the weakest machine in our toolkit, since it has a mobile processor.